examples/eks/eks-cluster.yaml

```diff
@@ -0,0 +1,62 @@
+---
+apiVersion: eksctl.io/v1alpha5
+kind: ClusterConfig
+
+metadata:
+  name: <eks_cluster_name>
+  region: <eks_region>
+
+nodeGroups:
+  - name: operator-pool
+    instanceType: t3.small
+    desiredCapacity: 1
+    taints:
+      role: "operators:NoSchedule"
+
+  - name: scylla-pool
+    instanceType: i3.2xlarge
+    desiredCapacity: 3
+    labels:
+      pool: "scylla-pool"
+    taints:
+      role: "scylla-clusters:NoSchedule"
+    ssh:
+      allow: true
+    preBootstrapCommands:
+      - cpupower frequency-set -g performance
+      - curl -o /etc/yum.repos.d/scylla.repo -L http://repositories.scylladb.com/scylla/repo/3e3ec6ec-92b2-46ec-b64f-5b2338bc47de/centos/scylladb-4.0.repo
+      - yum install -y scylla-server hwloc ethtool
+      - /opt/scylladb/scripts/scylla_setup --ntp-domain amazon --no-coredump-setup --no-sysconfig-setup --no-raid-setup --no-io-setup --no-bootparam-setup --no-ec2-check --no-cpuscaling-setup
+      - /opt/scylladb/scripts/scylla_raid_setup --disks /dev/nvme0n1 --root /mnt/raid-disks/disk0 --update-fstab
```

Comment:
This only works for the selected instances - should be documented. Smaller instances would not have NVMe and bigger instances would have 2 or more NVMe.

Reply:
I think this is acceptable since the setup for scylla is fixed to be 3 x i3.2xlarge